pages/contracts/cw1/subkeys/execute.tsx

```diff
@@ -85,33 +93,59 @@ const CW1SubkeysExecutePage: NextPage = () => {
     subtitle: 'Address of the CW1 Subkeys contract',
   })
 
-  const messageState = useInputState({
-    id: 'message',
-    name: 'message',
-    title: 'Message',
-    subtitle: 'Message to execute on the contract',
-    defaultValue: JSON.stringify({ key: 'value' }, null, 2),
-  })
-
   const recipientState = useInputState({
     id: 'recipient-address',
     name: 'recipient',
     title: 'Recipient Address',
     subtitle: 'Address of the recipient',
   })
 
-  const showAmountField = type && isEitherType(type, ['increase_allowance', 'decrease_allowance'])
+  const validatorState = useInputState({
+    id: 'validator-address',
+    name: 'validator',
+    title: 'Validator Address',
+    subtitle: 'Address of the validator',
+  })
+
+  const dstValidatorState = useInputState({
+    id: 'validator-address',
+    name: 'validator',
+    title: 'DST Validator Address',
+    subtitle: 'Address of the dst validator',
+  })
+
+  const showAmountField =
```

Comment:
Amount shows up even tough it's not supposed to under set-permissions, update-admins and so on
![Screen Shot 2022-05-31 at 10 07 25](https://user-images.githubusercontent.com/16416963/171113144-9e095631-7775-4dc7-bec0-3526769f5d3f.png)
![Screen Shot 2022-05-31 at 10 07 20](https://user-images.githubusercontent.com/16416963/171113153-4b8b4b7e-10ad-4fff-bd0a-a65b06d03aab.png)